### PR TITLE
Migrated some fragments from Java to Kotlin

### PR DESCRIPTION
**Pull request by AlecsisDuarte**
Migrated the Answer, QuestionsByActivity & Search fragments from java to kotlin, and also upgraded some dependencies due to issues while building the APK

### CODE REVIEW

File: app/src/main/java/com/josycom/mayorjay/flowoverstack/ui/fragment/QuestionsByActivityFragment.kt [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+import androidx.paging.PagedList
+import androidx.recyclerview.widget.DefaultItemAnimator
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.josycom.mayorjay.flowoverstack.R
+import com.josycom.mayorjay.flowoverstack.adapters.QuestionAdapter
+import com.josycom.mayorjay.flowoverstack.databinding.FragmentQuestionsByActivityBinding
+import com.josycom.mayorjay.flowoverstack.model.Question
+import com.josycom.mayorjay.flowoverstack.ui.activity.AnswerActivity
+import com.josycom.mayorjay.flowoverstack.util.AppConstants.*
```
Comment (mayorJAY) [inline app/src/main/java/com/josycom/mayorjay/flowoverstack/ui/fragment/QuestionsByActivityFragment.kt:20]: Avoid wild card imports

**Comment by mayorJAY** [comment]
Did you try running the App after the change? What was your experience like?

File: app/src/main/java/com/josycom/mayorjay/flowoverstack/ui/fragment/QuestionsByActivityFragment.kt (hunk 1 above)
Comment (AlecsisDuarte) [reply app/src/main/java/com/josycom/mayorjay/flowoverstack/ui/fragment/QuestionsByActivityFragment.kt:20]: Yeah, totally agree

**Comment by AlecsisDuarte** [comment]
> Did you try running the App after the change? What was your experience like?

Yes, I was able to run the application successfully and didn't find any apparent issue 🎉

**Comment by AlecsisDuarte** [comment]
@mayorJAY - Done!

**Comment by mayorJAY** [comment]
LGTM

**Review by mayorJAY — APPROVED** [review]

**Pull request merged**